### DISCUSSION TITLE
Jellyseerr Something went wrong while trying to sign in

### DISCUSSION THREAD

**Discussion by iJasonWade** [seerr-team/seerr#420] [Q&A] [answered]
After running perfectly for months my Jellyseerr instance all of a sudden acted like I had to setup the server again and gives me this message when I try to set it up:
![Xnip2023-07-02_23-33-16](https://github.com/Fallenbagel/jellyseerr/assets/12824479/1ca3d084-44f7-48a2-b953-0624264b3aa9)

Nothing in the logs shows an error and I also see that according to the logs it is syncing with my Jellyfin instance but it does show an invalid token error in the logs for Jellyfin.

I tried deleting the `db/` files and the `settings.json` file but it still will not let me past the initial screen.

**Comment by fallenbagel** [comment, marked as answer]
Can you hop over to discord so we can follow through? I have a few troubleshooting steps I want you to try out

**Reply by nicolasestrem** [reply]
So what was the answer? That's kind of the whole point of using git; make the solution visible.

**Reply by gauthier-th** [reply]
The whole point of git is to publish our source code and track our code-related issues, not to do support when something is misconfigured by users.
Feel free to hop onto Discord, we can provide support there.

**Reply by fallenbagel** [reply]
We have support channels separately from github. The reason we don't use gh for support is the disconnect from real-time conversations, which makes it extremely frustrating to troubleshoot. And it's just the two of us not a whole team who can reply to support questions. We even mention it on readme to hop onto discord for support. The whole point of github is to publish our source code and track actual issues (bugs and feature requests and prs and occasional questions some people have regarding the code/repo/project/etc).

**Reply by fallenbagel** [reply]
This discussion is **2 YEARS OLD** and not relevant now. Especially since we are on v2 now. And im pretty sure it does not apply to your issue, even if it seemed similar.

**Reply by fallenbagel** [reply]
And their issue was network related.
https://discord.com/channels/952656177924300932/1125487726804865156
